src/types.ts

```diff
@@ -69,3 +92,22 @@ export type ToastOptions = {
   from?: "top" | "bottom";
   layout?: Layout;
 };
+
+export type CustomToastOptions = Omit<BaseToastOptions, "preset"> & {
+  /**
+   * Defaults to `done`.
+   */
+  preset?: "custom"; // TODO custom option
+  /**
+   * The name of an iOS-only SF Symbol. For a full list, see https://developer.apple.com/sf-symbols/.
+   * @platform ios
+   */
+  iosIconName?: string;
```

Comment:
Same note about `icon` from above.

Reply:
yeah, I thought too, api will be the same with `zeego`, I will do that! 
btw I think we can add the same push `width` and `height` into the `ios` object, will be good! 
just like: 
```
toast({ preset: 'custom', icon: { ios: { name, color, width, height } } })
```

Reply:
I think that's a good idea.

Reply:
>I think we can add the same push width and height into the ios object, will be good!

Oh, I just noticed we can't do that since it also needs to use for "done" or "error" cases. 
so will keep it.